src/main/java/org/jvnet/hudson/plugins/platformlabeler/PlatformDetailsTask.java

```diff
@@ -226,15 +233,24 @@ private String checkLinux32Bit(@NonNull final String arch) {
     PREFERRED_LINUX_OS_NAMES.put("ol", "OracleServer");
     PREFERRED_LINUX_OS_NAMES.put("opensuse", "openSUSE");
     PREFERRED_LINUX_OS_NAMES.put("ubuntu", "Ubuntu");
+    PREFERRED_LINUX_OS_NAMES.put("scientific", "Scientific");
+    PREFERRED_LINUX_OS_NAMES.put("Scientific Linux", "Scientific");
```

Comment:
The preferred names should be mappings from the values returned by the id gathering functions.  Does one of the ID gathering functions return "Scientific Linux"?
```suggestion
```

Reply:
Yes in the redhat-release files for Scientific return Scientific Linux